sui/transfer-object.js

```diff
@@ -0,0 +1,77 @@
+const { TransactionBlock } = require('@mysten/sui.js/transactions');
+const { Command, Option } = require('commander');
+const { loadConfig, printInfo, isKeccak256Hash, isString } = require('../evm/utils');
+const { addBaseOptions } = require('./cli-utils');
+const { getWallet } = require('./sign-utils');
+const { loadSuiConfig } = require('./utils');
+
+async function transferCap(options) {
+    const config = loadSuiConfig(options.env);
+    const [keypair, client] = getWallet(config.sui, options);
+    const tx = new TransactionBlock();
+
+    tx.transferObjects([`${options.objectId}`], tx.pure(options.recipient));
+
+    const result = await client.signAndExecuteTransactionBlock({
+        transactionBlock: tx,
+        signer: keypair,
+        options: {
+            showObjectChanges: true,
+            showBalanceChanges: true,
+            showEvents: true,
+        },
+    });
+    printInfo('Transaction result', JSON.stringify(result));
+}
+
+async function getConfigAndTransferObject(options) {
+    const objectId = loadConfig(options.env).sui.contracts[`${options.contractName}`].objects[`${options.objectName}`];
```

Comment:
not very readable, split into multiple lines, use `?` operator to handle undefined cases, and throw error. Also, config loading is being duplicated. No reason to have transferCap in a different function, while this just retrieves objectId from config. See the `processCommand` pattern from other scripts.